File: C#/백준/Bronze/23348. 스트릿 코딩 파이터/스트릿 코딩 파이터.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(String[] args){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int a = int.Parse(Console.ReadLine());
            int[] n = new int[a];
            for (int i=0;i<a;i++){
                int total =0;
                for (int j=0;j<3;j++){
                    int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                    total+=aa[0]*bb[0]+aa[1]*bb[1]+aa[2]*bb[2];
                }
                n[i] = total;
            }
            Console.WriteLine(n.Max());
        }
    }
}

File: Python/백준/Bronze/23348. 스트릿 코딩 파이터/스트릿 코딩 파이터.py
a,b,c = map(int,input().split())
n = int(input())
aa = []
for i in range(n):
    total = 0
    for j in range(3):
        d,e,f = map(int,input().split())
        total+=a*d+b*e+c*f
    aa.append(total)
print(max(aa))